devel/ansible/roles/dev/files/motd

```diff
@@ -3,6 +3,7 @@ Welcome to the Bodhi development environment! Here are some helpful commands:
 
 bdocs:      Build Bodhi's documentation.
 blog:       View Bodhi's log.
```

Comment:
Perhaps instead of adding ```btail```, we could add something like this to this line "(Hint: this accepts journalctl's flags.)